analysis/tests/src/expected/Completion.res.txt

```diff
@@ -1381,3 +1383,15 @@ posCursor:[355:23] posNoWhite:[355:22] Found expr:[0:-1->355:23]
 posCursor:[355:23] posNoWhite:[355:22] Found expr:[355:12->355:23]
 []
 
+Complete tests/src/Completion.res 362:8
+posCursor:[362:8] posNoWhite:[362:7] Found expr:[360:8->365:3]
+posCursor:[362:8] posNoWhite:[362:7] Found expr:[361:2->365:3]
+XXX Pexp_match with 1 cases not handled
+[]
+
+Complete tests/src/Completion.res 367:30
+posCursor:[367:30] posNoWhite:[367:29] Found expr:[367:11->367:30]
+posCursor:[367:30] posNoWhite:[367:29] Found expr:[367:16->367:30]
+XXX Pexp_match with 1 cases not handled
```

Comment:
This shows that parser recovery also gives 1 case when no more cases others follow.